dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -5948,21 +5948,21 @@ BEGIN
     6827 "Semnul tai tham satkaankuu"
     6828 "Semnul tai tham hang"
     6829 "Semnul tai tham caang"
-    6832 "Combining Doubled Circumflex Accent"
-    6833 "Combining Diaeresis-Ring"
-    6834 "Combining Infinity"
-    6835 "Combining Downwards Arrow"
-    6836 "Combining Triple Dot"
-    6837 "Combining X-X Below"
-    6838 "Combining Wiggly Line Below"
-    6839 "Combining Open Mark Below"
-    6840 "Combining Double Open Mark Below"
-    6841 "Combining Light Centralization Stroke Below"
-    6842 "Combining Strong Centralization Stroke Below"
-    6843 "Combining Parentheses Above"
-    6844 "Combining Double Parentheses Above"
-    6845 "Combining Parentheses Below"
-    6846 "Combining Parentheses Overlay"
+    6832 "Accent circumflex dublu îmbinativ"
```

Comment:
I didn't intend to suggest changing "dublat" (which was fine) into "dublu", but to express things more in line with each other. A few lines below there is "Paranteză îmbinativă acoperitoare", and what I thought that could be improved here was something into "Accent îmbinativ circumflex dublat". The same kind of suggestion was also for the following lines.
I'm writing this comment just to clarify my initial idea. This is not, however, a request for change.